x/ibc/core/03-connection/types/keys.go

```diff
@@ -29,31 +29,21 @@ const (
 	ConnectionPrefix = "connection-"
 )
 
+// IsValidConnectionID checks if a connectionID is in the format required for parsing connection
+// identifier. The connection identifier must be in the form: `connection-{N}
```

Comment:
Note: I believe we only constrain our own chain's connection identifiers this way right. Worth documenting this fact clearly. Ditto for client and channel id's